support-workers/src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -41,8 +44,18 @@ class SendThankYouEmail(thankYouEmailService: EmailService, servicesProvider: Se
     case _ => Future.successful(None)
   }
 
+  def getProductRatePlanId(product: ProductType, isTestUser: Boolean) = {
+    val touchpointEnvironment = TouchPointEnvironments.fromStage(Configuration.stage, isTestUser)
+    (product match {
+      case d: DigitalPack => d.productRatePlan(touchpointEnvironment)
+      case p: Paper => p.productRatePlan(touchpointEnvironment)
```

Comment:
Do we need this if `product` is already the correct ADT, or is it because we added `produxtRatePlan`.